### PR TITLE
syslog-ng-debun: maintenance updates and small fixes

### PR DESCRIPTION
**Pull request by jszigetvari**
* version bump
    * updated copyright information
    * added option and functionality to save private keys
      (upon express request via a CLI option (-K))
    * the script now removes Splunk auth tokens from the
      config by default
    * minor rework on the running config collection
    * added CLI option to override tcpdump CLI options
    * print a warning if SELinux is in enforcing mode
    * added support for gathering the installed SCL include files
    * added options for cpio to dereference symlinks when copying
    * fixed cpio related portability problems
    * added support for gathering WEC PID and lsof information
    * fixed ps related portability issues
    * the script now gathers statistics at the beginning and end
      of the debug session

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by Kokan** [comment]
@kira-syslogng ok to test

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: contrib/syslog-ng-debun [hunk 1]
```diff
@@ -707,7 +708,7 @@ acquire_syslog_pids () {
 		sngpid=$( cat ${piddir}/syslog-ng.pid )
 	else
 		# Handle when fhs is "linux"-like
-		sngpid=$( cat ${piddir}/syslog-ng.pid )
+		sngpid=$( $pseao pid,args | grep "[s]yslog-ng " | head -1 | while read PID CMD; do echo $PID; done )
```
Comment (furiel) [inline contrib/syslog-ng-debun]: Me feels using syslog-ng.pid file safer. Why did you need to use ps?

Comment (jszigetvari) [reply contrib/syslog-ng-debun]: @furiel I decided in favor of this approach due to the fact that I had to use this approach when dealing with the auto-detection of pids for the running WEC instances. At the same time, it is also very well possible that there may be multiple syslog-ng instances running in parallel (the same may be also true in case of wec), and the old pidfile approach only showed one syslog-ng process from the many ones possible.
The added benefit of this approach is that this way the pid detection is done in the very same way in case of both wec and syslog-ng.

**Comment by jszigetvari** [comment]
Please wait a few extra days before merging this.
I have discovered a few glitches during my tests. Please stand by as I try to resolve them.

**Comment by jszigetvari** [comment]
@jszigetvari The bugs have been fixed.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by jszigetvari** [comment]
Please suspend the merging of this PR once more, as I realized that I have to fix the find commands as well in order to follow symlinks.

**Comment by jszigetvari** [comment]
I committed my latest changes. All things should work this time.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS
